package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@yext/search-ui-react",
-  "version": "1.6.7",
+  "version": "1.6.8",
   "description": "A library of React Components for powering Yext Search integrations",
   "author": "slapshot@yext.com",
```

Comment:
Maybe should be a separate change, but shouldn't this say watson@yext.com?

Reply:
I figure we might as well do it here, since otherwise it'd be yet another patch version lol.

Reply:
True

Reply:
Added in [new commit](https://github.com/yext/search-ui-react/pull/480/commits/ae9dcbef20f78a9045160407d5b7d682cc792b28)